whylog/front/utils.py

```diff
@@ -0,0 +1,41 @@
+class FrontInput(object):
+    """
+    Object of this class represents line in the file.
+    """
+
+    def __init__(self, offset, line_content, resource_location):
+        """
+        Basic constructor of FrontInput.
+        :param offset: byte of the first character of line in the file.
+        :param line_content: the content of the line.
+        :param resource_location: obeject of ResourceLocation.
+        """
+        self.offset = offset
+        self.line_content = line_content
+        self.resource_location = resource_location
+
+
+class AbstractResourceLocation(object):
+    """
+    The Abastract class of for agregating informations about
+    file where are logs stored.
+    """
+
+    def __init__(self, path, host):
+        self.path = path
+        self.host = host
+
+    def __repr__(self):
+        return '%s:%s' % self.host, self.path
+
+
+class LocallyAccessibleLogOpener(AbstractResourceLocation):
+    """
```

Comment:
I'm not sure if it really will open the logs. The @kgromadzki told me to write this class and it will be used in Config. I should ask him what this class really will do.